tests/unit/linalg/test_vector.cpp

```diff
@@ -15,15 +15,17 @@
 
 using namespace mfem;
 
-TEST_CASE("Vector init-list construction", "[Vector]")
+TEST_CASE("Vector init-list and C-style array constructors", "[Vector]")
 {
    real_t ContigData[6] = {6.0, 5.0, 4.0, 3.0, 2.0, 1.0};
    Vector a(ContigData, 6);
    Vector b({6.0, 5.0, 4.0, 3.0, 2.0, 1.0});
```

Comment:
☝️  Here without `.0`.

Reply:
Done in c232029d84270e0ec0fee02aea5efb7e3719ff7d.